swim.go

```diff
@@ -45,6 +49,7 @@ type Config struct {
 }
 
 type SWIM struct {
+	ID string
```

Comment:
In my case( #96 ) I use `Member`
```go
// Information of this node
	member *Member
```
I think it is more useful because Member has IP address, Incarnation number, also node's ID.
even if `Member` Name is ambiguous to express myself, i think it is okay to do it with proper naming of variables.

Reply:
I agree with leesd556